theano/gpuarray/basic_ops.py

```diff
@@ -319,7 +319,9 @@ def c_header_dirs(self):
     def _generate_kernel_code(self, k):
         code = '\\n'.join(l for l in k.code.split('\n'))
         code = code.replace('"', '\\"')
-        return ("""static const char *%(cname)s = "%(code)s";""" %
+        return ("""static const char *%(cname)s_unsigned = "%(code)s";
+                static const char *%(cname)s = (char *)%(cname)s_unsigned;
```

Comment:
I understand.
My point is that `%(code)s` here is unlikely to have values > 128, but `%(bcode)s` that was defined in `_generate_kernel_bin` definitely had them (but it was removed in #5847).
Since it could still happen in `%(code)s`, I'm OK with the change.